packages/docs/pages/ui-onboarding/index.md

```diff
@@ -80,7 +80,38 @@ The naming convention follows env-{beta/stable}
 
 Many teams want alignment between environments on the platform, so we allow a .custom\_release.sh file inside of your .travis directory in your dev repo.
 
-You can set up any push logic you&#39;d like to in this repo. Many times, teams will align ci/qa together, so an example of that would be: ![](RackMultipart20220401-4-1ogbjj7_html_3d8d7472659d9099.png)
+You can set up any push logic you&#39;d like to in this repo. Many times, teams will align ci/qa together, so an example of that would be: 
+
+  #!/bin/bash
```

Comment:
Can you wrap it in the ode block so it renders nicely into a code editor component?
```
```sh
content
```
```